render/container.go

```diff
@@ -216,6 +216,12 @@ func MapContainer2IP(m report.Node, _ report.Networks) report.Nodes {
 		return report.Nodes{}
 	}
 
+	// if this container doesn't make connections, we can ignore it
+	_, doesntMakeConnections := m.Latest.Lookup(kubernetes.DoesNotMakeConnection)
```

Comment:
If the "DoesNotMakeConnection", bit is more generic, it's a bit odd it lives in the `kubernetes` probe package...